numpy/core/einsumfunc.py

```diff
@@ -884,39 +886,68 @@ def einsum(*operands, **kwargs):
     -----
     .. versionadded:: 1.6.0
 
+    The Einstein summation convention can be used to compute
+    many multi-dimensional array operations. Additionally, the explicit
+    mode of this function provides further flexibility that can be used
+    to compute other array operations that might otherwise not be
+    considered classical Einstein summation operations. `einsum`
+    provides a succinct way of representing many array operations in a
+    clear and consistent format.
+
+    A non-exhaustive list of array operations which can be computed by
+    `einsum` is shown below along with examples,
+
+        * Trace of an array.
+        * Return a diagonal.
+        * Array axis summations.
+        * Transpositions and permutations.
+        * Matrix multiplication and vector dot product.
+        * Broadcasting and scalar multiplication.
+        * Tensor contractions.
+        * Chained array operations, in efficient calculation order.
```

Comment:
"array operations" is pretty vague.

Also, does this list render correctly in sphinx?

Reply:
Thanks for the overall feedback, Eric.

I agree it is vague, to be consistent with other paragraphs and for clarity perhaps "multi-dimensional, linear algebraic array operations" is better.

Sphinx rendering of bullet points is OK. I have copied the format from other numpy docstrings and tested against an online sphinx renderer.

Reply:
It will render without a vertical bar (quote mark) if you untab four spaces, aligning the `*` with the previous text. Also IMO `examples:` not `examples,`

Reply:
adopted